.github/ISSUE_TEMPLATE/feature_request.md

```diff
@@ -0,0 +1,24 @@
+---
+name: "\U0001F680 Feature Request"
+about: "I have a specific suggestion for the TypeScript and HTML API of Vaadin components!"
+title: ''
+labels: 'i: needs triage'
+assignees: ''
+
+---
+
+### Describe a problem you proposal is addressing
+
+A concise description of what the problem or reason is. For example, I'm always frustrated when [...]
```

Comment:
```suggestion
A concise description of why you are proposing a change. For example, you would like a new feature to solve a use-case, or you want an existing feature changed because it is difficult to use.
```
Would like this to not focus on frustrations.